README.md

```diff
@@ -57,6 +57,10 @@ If Optimize, Identity, and Keycloak are not needed you can use the [docker-compo
 docker-compose -f docker-compose-core.yaml up -d
 ```
 
+> **Note**
```

Comment:
I would propose to not just put this as a node, but create a section before Camunda 7, like `Development` which describes that there is a `dev` profile which an be enabled by adding `--profile dev` to your docker compose command. And then also describe what it does, i.e. spin up Kibana and Kibana is accessible under `http://localhost:5601`. Otherwise we assume or have people google how to specify a profile, or how to access kibana.

Also I would propose to just add the container also to the full compose file using the dev profile, to keep the files in sync and the explanation applicable for both setups.